BetterAPI.csproj

```diff
@@ -1,37 +1,37 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
     <AssemblyName>BetterAPI</AssemblyName>
     <RootNamespace>BetterAPI</RootNamespace>
   </PropertyGroup>
 
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
+    <Exec Command="xcopy &quot;$(TargetPath)&quot; &quot;D:\SteamBackup\steamapps\common\Risk of Rain 2\BepInEx\plugins&quot; /Y" />
+  </Target>
+
   <ItemGroup>
     <Reference Include="Assembly-CSharp">
-      <HintPath>..\Libs\Assembly-CSharp.dll</HintPath>
+      <HintPath>D:\Mod Workspace\RoR2Libs\RoR2Libs\Publicized\Assembly-CSharp.dll</HintPath>
```

Comment:
Uh oh, I need to find a way to deal with this, for now I think we need to just undo this for this commit and then find a solution

Reply:
Okay I think what I recommend here for the future, is that you put a symlink at `..\Libs` that points to where you have your libraries stored. It's a little hacky but it should work?

Reply:
Or just don't stage those changes next time so you can have it changed locally